src/containers/settings/Settings.tsx

```diff
@@ -16,13 +20,35 @@ const Settings: React.FC = () => {
       </Helmet>
       <div className="content-container">
         <Title>Settings</Title>
-        <Form name="basic" onFinish={onFinish}>
+        <Form name="basic">
```

Comment:
This need a todo to actually hook it up to something?

Reply:
I removed it and made an issue